afs/media/js/views/components/reports/group.js

```diff
@@ -14,14 +14,15 @@ define([
             params.configKeys = ['tabs', 'activeTabIndex'];
             Object.assign(self, reportUtils);
             self.sections = [
-                {id: 'name', title: 'Names and Classifications'},
-                {id: 'existence', title: 'Existence'},
-                {id: 'events', title: 'Events'},
+                {id: 'name', title: 'Names, Identifiers, Classifications'},
+                {id: 'description', title: 'Description'},
+                {id: 'existence', title: 'Timeline'},
                 {id: 'location', title: 'Location'},
                 {id: 'parthood', title: 'Parthood'},
```

Comment:
This parthood tab needs to go away/ be absorbed by another tab.  I would say for the time being just place the component into the Larger Entities tab.